frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/EditTab/OptionListEditor/ManualOptionsEditor/ManualOptionsEditor.tsx

```diff
@@ -0,0 +1,57 @@
+import { useOptionListEditorTexts } from '@altinn/ux-editor/components/config/editModal/EditOptions/OptionTabs/hooks';
+import type { Option } from 'app-shared/types/Option';
+import classes from './ManualOptionsEditor.module.css';
+import type { IGenericEditComponent } from '@altinn/ux-editor/components/config/componentConfig';
+import type { SelectionComponentType } from '@altinn/ux-editor/types/FormComponent';
+import { useTranslation } from 'react-i18next';
+import React, { useRef } from 'react';
+import { StudioCodeListEditor, StudioModal, StudioProperty } from '@studio/components';
+
+type ManualOptionsEditorProps = Pick<
+  IGenericEditComponent<SelectionComponentType>,
+  'component' | 'handleComponentChange'
+>;
+
+export function ManualOptionsEditor({
+  component,
+  handleComponentChange,
+}: ManualOptionsEditorProps): React.ReactNode {
+  const { t } = useTranslation();
+  const modalRef = useRef<HTMLDialogElement>(null);
+  const editorTexts = useOptionListEditorTexts();
+
+  const handleBlurAny = (options: Option[]) => {
+    if (component.optionsId) {
+      delete component.optionsId;
+    }
```

Comment:
Dette bør være en funksjon på utsiden av komponenten. Og den bør helst være ren - det vil si at den returnerer et nytt objekt i stedet for å manipulere inndataobjektet.

Reply:
Dette er kode som er flyttet bare. Og det er kode som vil endres i større grad i [denne PRen](https://github.com/Altinn/altinn-studio/pull/14276), så jeg vet ikke hvor hensiktsmessig det er å gjøre det her? 🤔

Reply:
Ok, vi lar det ligge, så unngår vi konflikter.

Reply:
Da kan sikkert jeg se på å gjøre det om til en generell funksjon i [PRen](https://github.com/Altinn/altinn-studio/pull/14276) nevnt ovenfor